### PR TITLE
Refactor/adds new company to hmr

### PR DESCRIPTION
**Pull request by thaiscristine**

### CODE REVIEW

File: futurice-tribes/src/Futurice/Company.hs [hunk 1]
```diff
@@ -88,6 +89,9 @@ companyFuturiceGmbH = fromMaybe (error "Company Futurice GmbH") $ companyFromTex
 companyFutuSwedenAB :: Company
 companyFutuSwedenAB = fromMaybe (error "Company FutuSweden AB") $ companyFromText "Futu Sweden AB"
 
+companyFuturiceHubsOy :: Company
```
Comment (thaiscristine) [inline futurice-tribes/src/Futurice/Company.hs:92]: @mixman Are these lines needed at all? Or only the .json file from hmrs is enough to add new company data?

Comment (mixman) [reply futurice-tribes/src/Futurice/Company.hs:92]: JSON should be enough. I didn't pinpoint immediate use-case. Some of these could be related to some UI component / WIP.

**Comment by lumicake** [comment]
You will need to update the sha256 sum for `data/tribes.json` if you add the new tribe/department that was specified for Hubs.

**Comment by thaiscristine** [comment]
> You will need to update the sha256 sum for `data/tribes.json` if you add the new tribe/department that was specified for Hubs.

Good catch. Updated 👌🏽

**Review by JoshStrobl — APPROVED** [review]
LGTM, thanks!

**Review by lumicake — APPROVED** [review]
LGTM :rocket:

**Pull request merged**